frontend/benefit/handler/src/components/applicationForm/formContent/attachmentsList/useAttachmentsList.ts

```diff
@@ -14,13 +14,14 @@ type ExtendedComponentProps = {
   attachments: [];
   isRemoving: boolean;
   isUploading: boolean;
-  handleRemove: (attachmentId: string) => void;
+  handleRemove: (attachmentId: string, name: string) => void;
```

Comment:
The signature of the actual returned function didn't change; there's no name parameter there. The name is resolved inside that function by using the ID alone.